README.md

```diff
@@ -35,9 +35,10 @@ Access design documentation specific to the KNI and KubeVirt features in OpenShi
 ### KubeVirt (Virtualization in OpenShift)
 - [Clone VM](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/clone-vm/clone-vm)
 - Create VM
-- Expose Secret, CM and SA as Disks
+- [Expose Secret, CM and SA as Disks](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/expose-secret/expose-secret)
```

Comment:
```suggestion
```

Could you either:

1. Delete this line and add it to #193, or
2. Keep this line and update the title of this PR to something like "Add links for Exposing Secrets and PVC/VM Disk Link"?